### ISSUE TITLE
rio edit on a templated service should work

### ISSUE THREAD

**Issue by izaac** [rancher/rio#832]
**Describe the bug**

An error occurs when trying to `rio edit` a service that was deployed using the --template flag during `rio run`
```
# Error updating content:
#    error unmarshaling JSON: while decoding JSON: json: cannot unmarshal bool into Go struct field templateFile.template of type v1.TemplateMeta
#
```

**To Reproduce**
1. `rio run -n template-test -p 80:8080 --template  https://github.com/rancher/rio-demo`
2. `rio edit template-test` (edit the port or version)
3. Error occurs.

**Expected behavior**

Edit should save the change and update the rio service/workload.

Type:
**Rio version**: `rio info`
```
rio v0.6.0-rc3
```

**Comment by izaac** [comment]
This is working in dev build from master branch. @StrongMonkey  @rancher-max

**Comment by izaac** [comment]
@StrongMonkey @rancher-max validated fixed in `rio v0.6.0-rc4`
